count_files/__main__.py

```diff
@@ -75,6 +75,13 @@
                          '(table) and for counting the total number of files ("-t" or "--total"). '
                          'This option disables it.')
 
+parser.add_argument('-ah', '--args-help', type=str, dest='argument',
```

Comment:
Question: Isn't it possible to make it like this?

```
count-files -h <keyword>
count-files --help <keyword>
````

Reply:
It is possible. To do this, you need to completely disable the standard parser help.
Then you can add custom optional argument `-h, --help`. But in this case, you can set (or I do not know how to do it without side effects) the default value for an optional argument if you specify action ="..." or make it positional argument. 
We already have one `path` positional argument with the default value, there will be a problem with the recognition of arguments.
You can make the default value with action = "", but then the `args. help` will be in the namespace even if it was not specified. And accordingly the function that processes it will be executed.
We can do `class CustomAction(argparse.Action)`.
I do not know whether this will be useful, but it can complicate the program.
https://docs.python.org/3.7/library/argparse.html#action  see: An example of a custom action. There you can manipulate the setattr in the namespace.
https://docs.python.org/3.7/library/argparse.html#argparse.Action

So, if you disable the standard help, then `--help` can not be used without keyword.

We have previously discussed:
https://github.com/victordomingos/Count-files/issues/87
I left the standard help, because the call `--help` without keyword is more familiar to the user.
If we implement: `count-files --help <keyword>`
and then do `--help` without keyword, then the standard error output of the parser will be:
```
usage: count-files [-h HELP] [-v] [-st] [-a] [-nr] [-c] [-nf] 
                   [-t EXTENSION] [-alpha] [-fe FILE_EXTENSION] [-p]
                   [-ps PREVIEW_SIZE] [-fs]
                   [path]
count-files: error: argument -h/--help: expected one argument
```
It will be unclear what argument expects help from us.

So, the argument help is a supplement to the standard help.